<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>311</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>772</width>
    <height>311</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>772</width>
    <height>311</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Senai Excel Management - MENU</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Users/Adm.Lab/Documents/Python - Excel with Tkinter/img/excel.ico</normaloff>../../../Users/Adm.Lab/Documents/Python - Excel with Tkinter/img/excel.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 0, 0);</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo_senai">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>521</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/logo-senai.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="my_name">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>271</y>
      <width>771</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>By: VÃ­ctor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_delete">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>200</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: white;
border: none;
border-radius: 4px;
}

QPushButton:hover{
background-color: rgb(148,148, 148);
border: none;
}</string>
    </property>
    <property name="text">
     <string>DELETAR DADOS (D)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_insert">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: white;
border: none;
border-radius: 4px;
}

QPushButton:hover{
background-color: rgb(148,148, 148);
border: none;
}</string>
    </property>
    <property name="text">
     <string>INSERIR DADOS (C)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_search">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: white;
border: none;
border-radius: 4px;
}

QPushButton:hover{
background-color: rgb(148,148, 148);
border: none;
}</string>
    </property>
    <property name="text">
     <string>PROCURAR DADOS (R)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>SENAI EXCEL MANAGEMENT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_update">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: white;
border: none;
border-radius: 4px;
}

QPushButton:hover{
background-color: rgb(148,148, 148);
border: none;
}</string>
    </property>
    <property name="text">
     <string>ATUALIZAR DADOS (U)</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
